spec/config/data.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { NODES, NodeConfig } from '../../common/config/data';
+import { RPCNode } from '../../common/libs/nodes';
+import { Validator } from 'jsonschema';
+import { schema } from '../../common/libs/validators';
+import 'url-search-params-polyfill';
+import EtherscanNode from 'libs/nodes/etherscan';
+import InfuraNode from 'libs/nodes/infura';
+import RpcNodeTestConfig from './RpcNodeTestConfig';
+
+const v = new Validator();
+
+const validRequests = {
+  address: '0x72948fa4200d10ffaa7c594c24bbba6ef627d4a3',
+  transaction: {
+    data: '',
+    from: '0x72948fa4200d10ffaa7c594c24bbba6ef627d4a3',
+    to: '0x72948fa4200d10ffaa7c594c24bbba6ef627d4a3',
+    value: '0xde0b6b3a7640000'
+  },
+  token: {
+    address: '0x4156d3342d5c385a87d264f90653733592000581',
+    symbol: 'SALT',
+    decimal: 8
+  }
+};
+
+const testGetBalance = async (n: RPCNode) => {
+  const data = await n.client.call(
```

Comment:
These functions would probably be cleaner if you used `.then` instead of async since its only a single call.